src/sentry/receivers/features.py

```diff
@@ -37,24 +38,21 @@
 #         complete=True)
 
 
-# @event_processed.connect(weak=False)
-# def record_event_processed(project, group, event, **kwargs):
-#     feature_slugs = []
+@event_processed.connect(weak=False)
+def record_event_processed(project, group, event, **kwargs):
+    feature_slugs = []
 
 #     # Platform
 #     if group.platform in manager.location_slugs('language'):
 #         feature_slugs.append(group.platform)
 
-#     elif event.data.get(get_interface('csp')):
```

Comment:
I'm removing this. It doesn't work. In fact, it never worked. Lines 44-46 always worked for csp, which is why the tests always passed.